services/api/db/models.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+from django.contrib.auth.models import BaseUserManager,AbstractBaseUser
+
+#  Custom User Manager
+class UserManager(BaseUserManager):
+  def create_user(self, email, name, password=None, password2=None):
+      """
+      Creates and saves a User with the given email, name and password.
+      """
+      if not email:
+          raise ValueError('User must have an email address')
+
```

Comment:
Should have a layer of check here to validate if password and password2 are same?

Reply:
Can we rename password2 to confirm_password for better readability.